platform/linux-generic/pktio/dpdk.c

```diff
@@ -338,8 +332,11 @@ static struct rte_mempool *mbuf_pool_create(const char *name,
 	struct rte_pktmbuf_pool_private mbp_priv;
 	struct rte_mempool_objsz sz;
 	unsigned int elt_size = pool_entry->dpdk_elt_size;
-	unsigned int num = pool_entry->num;
+	unsigned int num = pool_entry->num, populated = 0;
 	uint32_t total_size;
+	uint64_t page_size = odp_sys_huge_page_size(), offset = 0, remainder = 0;
```

Comment:
In theory, a system could be using multiple huge page sizes (e.g. 2M and 1G) and this reservation would not use the default size. So, it would be safer to read the page size from the SHM block.

Reply:
OK, switched to use `odp_shm_info()` in v2.